### DISCUSSION TITLE
Error when invoke @adonisjs/lucid

### DISCUSSION THREAD

**Discussion by NathaelB** [adonisjs/core#2013] [Help] [answered]
Good evening. I am writing this topic to ask you for help!
When creating a project under Adonis with a classic adonis v5 API structure, an error occurs ...

What the problem ?
---
My problem it's when `noce ace invoke @adonisjs/lucid` in my Terminal which returns me a error :
![image](https://user-images.githubusercontent.com/64804778/101268395-902b1f80-3762-11eb-85dd-50ee89c2a653.png)

**My package.json in my folder server**
![image](https://user-images.githubusercontent.com/64804778/101268419-cff20700-3762-11eb-9f01-955e13e03503.png)

**Comment by thetutlage** [comment, marked as answer]
Is this from a new project or after you try to upgrade dependencies?

**Reply by NathaelB** [reply]
It's a new project.

**Reply by thetutlage** [reply]
Feels weird. Coz we don't install `@adonisjs/fold` and `@adonisjs/ace` in new projects. I just gave it a try by running `npm init adonis-ts-app hello-world`

**Reply by NathaelB** [reply]
What should I do then?

**Reply by thetutlage** [reply]
It can be nice if you can share the command you used to create the app and share it the created app as a repo

**Reply by NathaelB** [reply]
Okay
